packages/frontend/src/components/Input/Buttons.js

```diff
@@ -10,3 +10,82 @@ export function FullWidthButton(props) {
 		</button>
 	);
 }
+
+export function HalfWidthButton(props) {
+	return (
+		<button
+			onClick={props.onClick}
+			className="w-[45%] bg-primary text-white p-4 rounded-normal text-normal"
+		>
+			{props.children}
+		</button>
+	);
+}
+
+export function SmallButton(props) {
+	return (
+		<button
+			onClick={props.onClick}
+			className="w-[20%] bg-primary text-white p-4 rounded-normal text-normal mx-auto"
+		>
+			{props.buttonText}
+		</button>
+	);
+}
+
+/**
+ * Full width textbox along with a copy link button on the right
+ * @param {string} props.buttonText - Text for the button
+ * @param {function} props.onClick - Function to be called when the button is clicked. The current value of the textbox will be passed as an argument
+ */
+export function CopyTextButton(props) {
+	return (
+		<div className="flex flex-row rounded-normal bg-secondary w-full text-normal">
+			<div className="flex items-center justify-start w-full text-gray_text p-4 rounded-normal text-normal">
+				{props.linkText}
```

Comment:
Can this be a more general name, maybe just `displayText`?
Also need to change usages.